recovery/builddata.h

```diff
@@ -0,0 +1,28 @@
+#ifndef BUILDDATA_H
+#define BUILDDATA_H
+
+#include <QString>
+#include <QStringList>
+
+class BuildData
+{
+public:
+    BuildData();
+    void read(const char * filename);
+    bool isEmpty(void);
+    void getVersionParts(void);
```

Comment:
Seems like this function isn't implemented??

Reply:
forgot to remove